### DISCUSSION TITLE
Codelist of field BT-165

### DISCUSSION THREAD

**Discussion by lillabeatak** [OP-TED/eForms-SDK#622] [Q&A] [answered]
The codelist of field BT-165 is not complete, because an "Other" category would be needed, e.g. we can not match natural person any of the current values

**Comment by YvesJo** [comment, marked as answer]
Hi,

> "An enterprise is ‘_any entity engaged in an economic activity, irrespective of its legal form_’".

> "It is the economic activity that is the determining factor, not the legal form.
> In practice, this means that the self-employed, family firms, partnerships and associations or any other entity that is regularly engaged in an economic activity may be considered as enterprises."

(cf. [User guide to the SME definition](https://ec.europa.eu/docsroom/documents/42921) p.11)

so the current list should be sufficient for the classification of any enterprise, including in the case of a natural person.
